code/grant/html_lab04/templates/index.html

```diff
@@ -0,0 +1,249 @@
+<!DOCTYPE html>
+<html lang="en">
+<!-- HEAD SECTION -->
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.1/font/bootstrap-icons.css">
+
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/devicons/devicon@v2.15.1/devicon.min.css">
+
+    <link rel="stylesheet" href="{{url_for('static', filename='styles.css')}}">
+    <title>Grant's Portfolio</title>
+</head>
+<!-- HEAD SECTION END -->
+<!-- NAVBAR SECTION -->
+
+<body>
+    <nav class="navbar navbar-expand-lg bg-dark navbar-dark" id=top-bar>
+        <div class="container-fluid">
+            <a href="file:///C:/Users/grant/Documents/class_liger/code/grant/html_lab03/templates/HOME.html"
+                class="navbar-brand">Grant's Portfolio</a>
+            <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbutton">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+
+
+            <div class="collapse navbar-collapse" id="navbutton">
+                <ul class="navbar-nav ms-auto">
+                    <li class="nav-item">
+                        <a href="file:///C:/Users/grant/Documents/class_liger/code/grant/html_lab03/templates/ABOUT-US.html"
```

Comment:
These links won't work since they link directly to a file on your computer.  If you need a placeholder you can instead add `href="#"` and it will appear as a link but non-functional.